handbook/communication/content_guidelines/actionable_language.md

```diff
@@ -0,0 +1,597 @@
+# Actionable language 
+
+The content in our products and messages shape how our users perceive and use our product to get things done. Our writing must help our users understand and take action.
+
+## Headings and subheadings
+
+Headings and subheadings in our interfaces have three goals:
+
+- Help users identify their **place**.
+- Help users understand how things are **organized**.
+- Help users predict what **actions** they can take.
+
+Some specific guidelines for writing headings and subheadings:
+
+- **Be concise.** Headings and subheadings should be scannable and use only as many words are needed to convey their meaning.
+- **Write in sentence case.** Sentence case makes it feel less stiff and formal.
```

Comment:
Can all sentence case info go into its own section in **Style & mechanics** (maybe in with **Capitalization**) vs being mentioned in several places in **Actionable language**?

Reply:
I'm hesitant to make this change, because the *Actionable language* content can be a one-stop reference for specific types of UI content, particularly for designers. If the sentence case info is moved to the *Style & mechanics* page, it disconnects it from the context, and designers would have to either remember it or reference two locations along the way. Does this change how you feel about it at all? Happy to keep discussing!

Reply:
Oh, yes that does make sense, I had forgotten that this wouldn't _just_ be a style guide. Would you be willing to do both, then? As a writer that would be the first place I look for that info.

Reply:
Yes, of course! That makes complete sense.